default.nix

```diff
@@ -193,6 +193,8 @@ let
 
       stream-benchmark = callPackage ./stream { };
 
+      travis = callPackage ./travis { };
```

Comment:
The name is a bit unfortunate. There is also Travis CI in upstream nixpkgs (which is also called travis). Right now we use a subset which avoids name collisions but I would prefer to keep names separate from upstream package names. How about `travis-analyzer` or `travis-md`?

Reply:
Switched to `travis-analyzer`.